rotkehlchen/tests/api/test_defi_protocols.py

```diff
@@ -0,0 +1,532 @@
+from typing import TYPE_CHECKING
+
+import requests
+
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response_with_result
+
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
+
+
+def test_defi_protocols_endpoint(rotkehlchen_api_server: 'APIServer') -> None:
+    """Test that the defi protocols endpoint works properly"""
+    response = requests.get(api_url_for(rotkehlchen_api_server, 'defimetadataresource'))
+
+    result = assert_proper_response_with_result(response)
+    assert result == [
+        {
+            'identifier': 'ygovfinancev1',
+            'name': 'ygov.finance (v1)',
+            'icon': 'iearn.png',
+        },
+        {
+            'identifier': 'ygovfinancev2',
+            'name': 'ygov.finance (v2)',
+            'icon': 'iearn.png',
+        },
+        {
+            'identifier': 'zlotfinance',
+            'name': 'zlot.finance',
+            'icon': 'zlotfinance.png',
+        },
+        {
+            'identifier': 'finnexus',
+            'name': 'FinNexus',
+            'icon': 'finnexus.svg',
+        },
+        {
+            'identifier': 'pickle_finance',
+            'name': 'Pickle Finance',
+            'icon': 'pickle.png',
+        },
+        {
+            'identifier': 'pickle_finance_farms',
+            'name': 'Pickle Finance \u2022 Farms',
+            'icon': 'pickle.png',
+        },
+        {
+            'identifier': 'pickle_finance_staking',
+            'name': 'Pickle Finance \u2022 Staking',
```

Comment:
why the name has this thing here?

Reply:
ah is the dot. Can you ask the frontend if this is good or will have any issue?

Reply:
@tewshi please can you confirm there's no issue with this?

Reply:
what does it look like, one sec.

Reply:
Like this:

Pickle Finance • Staking